tests/main/document-portal-activation/fake-document-portal.py

```diff
@@ -11,6 +11,7 @@
 OBJECT_PATH = "/org/freedesktop/portal/documents"
 DOC_IFACE = "org.freedesktop.portal.Documents"
 
+
 class DocPortal(dbus.service.Object):
     def __init__(self, connection, object_path, logfile, errorfile):
         super(DocPortal, self).__init__(connection, object_path)
```

Comment:
It's Python 3, so this should probably be just `super().__init__(connection, object_path)`